packages/loader/container-loader/src/utils.ts

```diff
@@ -35,7 +35,12 @@ export interface IParsedUrl {
 	version: string | null | undefined;
 }
 
-export function parseUrl(url: string): IParsedUrl | undefined {
+/**
+ * Utility api to parse the IResolvedUrl.url into specific parts like querystring, path to get
```

Comment:
Please add a warning that this function may not be compatible with any Url Resolver.  Unless I'm misunderstanding something and that's not the case.